src/preload/preload.ts

```diff
@@ -93,7 +91,7 @@ contextBridge.exposeInMainWorld("engine", engineApi);
 const gameApi = {
     // Content
     downloadGame: (version: string): Promise<void> => ipcRenderer.invoke("game:downloadGame", version),
-    getScenarios: ((version) => ipcRenderer.invoke("game:getScenarios", version)) as GetScenarios,
+    getScenarios: (version: string) => ipcRenderer.invoke("game:getScenarios", version),
```

Comment:
A few of these had explicit casts like this that I've removed. They weren't all accurate.